packages/dds/tree/src/simple-tree/objectNode.ts

```diff
@@ -154,6 +157,17 @@ export type InsertableObjectFromSchemaRecord<
 			}
 		>;
 
+/**
+ * Helper used to remove annotations from a schema record before calling InsertableObjectFromSchemaRecord.
+ *
+ * @remarks
+ * TODO figure out why InsertableObjectFromSchemaRecord can't be linked here
```

Comment:
InsertableObjectFromSchemaRecord is part of internalTypes, which makes it impossible to link (see not in top level index file about system types exported there to avoid issues with them being in internal types. One of the reasons listed is so they could be linked IIRC).

You can move InsertableObjectFromSchemaRecord out of internal types, or move the link into private remakrs to mitigate this.